### DISCUSSION TITLE
Add Reordering Functionality to Signal Connections

### DISCUSSION THREAD

**Discussion by JohnnyRouddro** [godotengine/godot-proposals#8925] [Editor]
If there is some kind of a reordering functionality (maybe drag and drop?) and the function calls maintain that order, it could solve some problems in a project like when a function changes a variable another function needs.

![signal_reordering](https://github.com/godotengine/godot-proposals/assets/30183816/4a3b0199-c708-4722-a522-8eca514e35b6)

**Comment by Mickeon** [comment]
Tangentially related to https://github.com/godotengine/godot-proposals/discussions/7738
